code/addons/docs/docs/theming.md

```diff
@@ -14,9 +14,9 @@ Storybook theming is the **recommended way** to theme your docs. Docs uses the s
 Supposing you have a Storybook theme defined for the main UI in `.storybook/manager.js`:
 
 ```js
-import { addons } from '@storybook/manager-api';
+import { addons } from 'storybook/manager-api';
```

Comment:
*From my understanding* we should **not** update docs for using this pattern yet, and we should do it in Storybook 9 only. Can you confirm?

Reply:
True, I changed it back.